test/Report.php

```diff
@@ -229,6 +230,11 @@ public function render($template, $data = array()) {
 		$template = Libraries::locate('test.templates', $template, array(
 			'filter' => false, 'type' => 'file', 'suffix' => ".{$config['format']}.php"
 		));
+
+		if($template === null) {
```

Comment:
Ah okay. I can fix that if you like.

Also, do you want the commits to be flattened or are you happy with the several commits being merged?